src_assets/common/assets/web/fetch.js

```diff
@@ -0,0 +1,9 @@
+export default async (url,config) => {
+    const response = await fetch(url, config);
+    console.log(response);
+    if(response.status == 401){
+        const event = new Event("sunshine:session_expire");
+        window.dispatchEvent(event);
+    }
+    return response;
+};
```

Comment:
```suggestion
};

```

If you use VS Code, I think you can adjust this behavior so it does it automatically.